inkscape/src/main/assembly/dist/lace_grid.inx

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <inkscape-extension xmlns="http://www.inkscape.org/namespace/inkscape/extension">
 
-	<_name>Lace Grid</_name>
+	<_name>Rectangular Grid</_name>

```

Comment:
I would have called it orthogonal, but most lace makers will understand this better.